libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -759,6 +759,14 @@
         "of": "af"
       }
     },
+    "calculationSteps": {
+      "topBarTitle": "Se akkumulerede tidsserier",
+      "breadcrumb": "({{gridAreaCode}}) {{gridAreaName}}",
```

Comment:
This is the same in both the danish and english translations, so perhaps it could be moved to the template directly?

Reply:
Because this is used across multiple places where the step is part of the breadcrumbs, I think it's suitable as a translation to avoid duplication.